RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -344,15 +346,7 @@ void PulseShapeFitOOTPileupCorrection::apply(const CaloSamples & cs,
    std::vector<float> fitParsVec;
 
    if(tstrig >= ts4Min_&& tsTOTen > 0.) { //Two sigma from 0
-     pulseShapeFit(energyArr, pedenArr, chargeArr, pedArr, gainArr, tsTOTen, fitParsVec, noiseADCArr);
-   }
-   else if((tstrig < ts4Min_||tsTOTen < 0.)&&(ts4Min_==0)){
-     fitParsVec.clear();
-     fitParsVec.push_back(0.);
-     fitParsVec.push_back(0.);
-     fitParsVec.push_back(0.);
-     fitParsVec.push_back(999.);
-     fitParsVec.push_back(false);
```

Comment:
ok, makes sense for chi2.

default time value of -9999 is probably OK (except for maybe some code that tried to make sense of the two options for E=0)